parcel.go

```diff
@@ -2,59 +2,115 @@ package main
 
 import (
 	"database/sql"
+	"fmt"
 )
 
 type ParcelStore struct {
 	db *sql.DB
 }
 
+// NewParcelStore создает новый экземпляр ParcelStore.
+// Принимает указатель на объект базы данных и возвращает новую структуру ParcelStore. ParcelStore содержит соединение с базой данных.
 func NewParcelStore(db *sql.DB) ParcelStore {
 	return ParcelStore{db: db}
 }
 
+// Add - метод объекта ParcelStore, добавляющий Parcel в соответствующую таблицу
+// Принимает объект Parcel, и возвращает Id добавленной записи и ошибку
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
 
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	res, err := s.db.Exec("INSERT INTO PARCEL (CLIENT, STATUS, ADDRESS, CREATED_AT) VALUES (:client, :status, :address, :createdAt)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("createdAt", p.CreatedAt))
+
+	if err != nil {
+		return 0, fmt.Errorf("an error occurred while adding the parcel. %v", err)
+	}
+
+	id, err := res.LastInsertId()
+
+	if err != nil {
+		return 0, fmt.Errorf("an error occurred while getting the last parcel number. %v", err)
```

Comment:
За оборачивание ошибки лайк, но тут нужен `%w` (wrap)
```suggestion
		return 0, fmt.Errorf("an error occurred while getting the last parcel number. %w", err)
```

Reply:
Далее по коду тоже нужно подправить